cypress/integration/M.E/team-cards-display-well(via link).spec copy.js

```diff
@@ -0,0 +1,8 @@
+import { showThatAllTeamCardsDisplayProperly } from "../../support/M.E/utils";
```

Comment:
this looks extra, should delete?

Reply:
I am not quite sure what you mean by "extra" @BradHN1.  The function is what actually checks if team cards are displaying properly. The rest is actually how you get to the teams page. So I am trying to do the check for the various ways you can get to the teams page, without having to write the same code twice.